src/corenfc.cs

```diff
@@ -144,9 +150,141 @@ interface NFCIso15693Tag : NFCTag {
 		void ReadMultipleBlocks (NFCIso15693ReadMultipleBlocksConfiguration readConfiguration, Action<NSData, NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("stayQuietWithCompletionHandler:")]
+		void StayQuiet (Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("readSingleBlockWithRequestFlags:blockNumber:completionHandler:")]
+		void ReadSingleBlock (RequestFlag flags, byte blockNumber, Action<NSData, NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("writeSingleBlockWithRequestFlags:blockNumber:dataBlock:completionHandler:")]
+		void WriteSingleBlock (RequestFlag flags, byte blockNumber, NSData dataBlock, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("lockBlockWithRequestFlags:blockNumber:completionHandler:")]
+		void LockBlock (RequestFlag flags, byte blockNumber, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("readMultipleBlocksWithRequestFlags:blockRange:completionHandler:")]
+		void ReadMultipleBlocks (RequestFlag flags, NSRange blockRange, Action<NSData [], NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("writeMultipleBlocksWithRequestFlags:blockRange:dataBlocks:completionHandler:")]
+		void WriteMultipleBlocks (RequestFlag flags, NSRange blockRange, NSData[] dataBlocks, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("selectWithRequestFlags:completionHandler:")]
+		void Select (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("resetToReadyWithRequestFlags:completionHandler:")]
+		void ResetToReady (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("writeAFIWithRequestFlag:afi:completionHandler:")]
+		void WriteAfi (RequestFlag flags, byte afi, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("lockAFIWithRequestFlag:completionHandler:")]
+		void LockAfi (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("writeDSFIDWithRequestFlag:dsfid:completionHandler:")]
+		void WriteDsfi (RequestFlag flags, byte dsfid, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("lockDFSIDWithRequestFlag:completionHandler:")]
+		void LockDfsi (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("getSystemInfoWithRequestFlag:completionHandler:")]
+		void GetSystemInfo (RequestFlag flags, Action<nint, nint, nint, nint, nint, NSError> completionHandler);
```

Comment:
`Action<nint, nint, nint, nint, nint, NSError>` makes my head spin

Reply:
oops, missed that one (so many ugly ones)